nilearn/signal.py

```diff
@@ -168,23 +166,31 @@ def _detrend(signals, inplace=False, type="linear", n_batches=10):
 def _check_wn(btype, freq, nyq):
     wn = freq / float(nyq)
     if wn >= 1.:
-        warnings.warn(
```

Comment:
Do you have any strong reason to remove from here and add below ?

Reply:
I need the value for wn to print it - I was fixing a bug in the warning message.

Reply:
What is the bug in the warning message ?

Reply:
The warning message does not reflect action taken inside.